InSource/Helpers/Constants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Sdl.Community.InSource.Helpers
+{
+	public class Constants
+	{
+		public readonly string NotificationGroupId = "b0261aa3-b6a5-4f69-8f94-3713784ce8ef";
```

Comment:
move this into notification related logic.

Reply:
Done. the NotificationGroupId was moved from the Constants class